http/src/request/guild/update_guild.rs

```diff
@@ -162,6 +162,11 @@ impl<'a> UpdateGuild<'a> {
     }
 
     /// Set the enabled features of the guild.
+    ///
+    /// Attempting to add or remove the "COMMUNITY" feature requires the
```

Comment:
Should we follow the style of the docs and write `COMMUNITY` instead?

Reply:
Done